### DISCUSSION TITLE
Hi guys, Stopwolf here!

### DISCUSSION THREAD

**Discussion by Stopwolf** [OpenMined/courses#52] [Introduce Yourself]
Hi everyone, my name's Sinisa Stanivuk, but you can call me Stopwolf (that's a direct translation of my lastname 😄 ). I'm a final year undergraduate student working part time as an ML enigineer in the financial sector.
I'm here because I'm trying to learn new concepts and maybe even write my bachelor thesis on something related to private AI 😅

**Comment by iamtrask** [comment]
Hi Stopwolf! Welcome to the community - best of luck in your bachelor's thesis. I still remember writing mine - it was a really influential time in my career - about when I decided I was really interested in neural networks. Glad I kept on running with it!

Hope the course can be helpful for you in your thesis!

**Comment by em-blue** [comment]
Hey @Stopwolf ! :) Welcome! If you're interested in finance, I think you'll enjoy the example of money laundering detection in the course. Hope you enjoy!
